luxjs/forms/form.js

```diff
@@ -17,7 +17,7 @@
     //      formFieldChange: triggered when a form field changes:
     //          arguments: formmodel, field (changed)
     //
-    angular.module('lux.form', ['lux.form.utils'])
+    angular.module('lux.form', ['lux.form.utils', 'ui.select'])
         //
```

Comment:
Remove``ui.select`` module dependency, this should be optional and added at application level.
If you put this module her, lux needs to load ``angular-ui-select`` which is not what we want.